esrally/mechanic/provisioner.py

```diff
@@ -61,18 +61,15 @@ def no_op_provisioner():
 
 
 def docker_provisioner(cfg, car, cluster_settings, target_root, node_id):
-    distribution_version = cfg.opts("mechanic", "distribution.version", mandatory=False)
-    ip = cfg.opts("provisioning", "node.ip")
-    http_port = cfg.opts("provisioning", "node.http.port")
-    rally_root = cfg.opts("node", "rally.root")
     node_name_prefix = cfg.opts("provisioning", "node.name.prefix")
-    preserve = cfg.opts("mechanic", "preserve.install")
 
     node_name = "%s-%d" % (node_name_prefix, node_id)
     node_root_dir = "%s/%s" % (target_root, node_name)
 
-    return DockerProvisioner(car, node_name, cluster_settings, ip, http_port, node_root_dir, distribution_version,
-                             rally_root, preserve)
+    rally_root = cfg.opts("node", "rally.root", default_value=paths.rally_root(), mandatory=False)
```

Comment:
I think the `default_value` should not be necessary here? Did you encounter a case where it was not set? If yes, this might indicate a bug.

Reply:
I don't remember what led to this change, but I agree that it must have been a bug due to misuse.